modules/nw-control-dns-records-public-managed-zone-gcp.adoc

```diff
@@ -24,14 +24,14 @@ $ oc whoami
 system:admin
 ----
 
-. Copy the value of service_account.json in gcp-credentials secret in a file encoded-gcloud.json by running the following command:
```

Comment:
These prompts can stay based on the contributing guidelines and can be added to all of the command prompts:

"The following guidelines go into more detail about specific requirements and recommendations when using code blocks:

If a step in a procedure is to run a command, make sure that the step text includes an explicit instruction to "run" or "enter" the command. In most cases, use one of the following patterns to introduce the code block:

- <Step description>, run the following command:
- <Step description>, enter the following command:"

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

Reply:
Addressed